packages/shared/routes/login/views/SelectProfile.svelte

```diff
@@ -44,16 +44,18 @@
     <Logo width="64px" logo="logo-firefly" classes="absolute top-20" />
     <div
         class="profiles-wrapper h-auto items-start justify-center w-full {!$mobile &&
-            'overflow-y-auto'} flex flex-row flex-wrap"
+            'overflow-y-auto'} flex flex-row flex-wrap space-x-20"
     >
         {#each $profiles as profile}
-            <div class="mx-4 mb-8">
+            <div class="mb-8">
                 <Profile
                     bgColor="blue"
                     onClick={handleContinueClick}
                     name={profile.name}
                     id={profile.id}
                     isDeveloper={profile.isDeveloperProfile}
+                    networkType={profile?.networkType || 'devnet'}
+                    networkProtocol={profile?.networkProtocol || 'shimmer'}
```

Comment:
Rather than typing out the string on the right hand side of this, can we use the NetworkProtocol enum and the NetworkType enum.

Also I think its better to use the [Nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here instead of the logical or operator, to check if left hand side is undefined or null.